utility/utils.py

```diff
@@ -2482,3 +2482,22 @@ def get_registry_info(registry):
         raise KeyError(
             "CephCI configuration yaml file does not have 'registries' section"
         )
+
+
+def get_custom_config_value(test_data, key_name):
```

Comment:
instead create method `is_unsecured_registry(test_data)` which will return true or false based on custom configs and update registry accordingly.

Reply:
sure, i will update